### PR TITLE
Update zh_cn.lang

### PR DESCRIPTION
**Pull request by 3TUSK**
Don't merge this yet until I get approval from @tartaricacid and @ETStarLight. Thanks for your understanding. 

Any other inputs are welcome.

### CODE REVIEW

File: src/main/resources/assets/twilightforest/lang/zh_cn.lang [hunk 1]
```diff
@@ -50,72 +116,119 @@ tile.TFFireJet.5.name=火焰喷射-激活
...
 tile.TFTowerDevice.13.name=激活的砷铅铁反应堆
 tile.TFTowerTranslucent.0.name=再现方块
 tile.TFTowerTranslucent.1.name=激活的再现方块
 tile.TFTowerTranslucent.2.name=建造的砷铅铁方块
 tile.TFTowerTranslucent.3.name=激活的建造的砷铅铁方块
 tile.TFTowerTranslucent.4.name=反建方块
 tile.TFTowerTranslucent.5.name=砷铅铁反应堆碎片
-tile.TFTowerTranslucent.6.name=伪黄金方块
-tile.TFTowerTranslucent.7.name=伪钻石方块
+tile.TFTowerTranslucent.6.name=伪黄金块
```
Comment (TartaricAcid) [inline src/main/resources/assets/twilightforest/lang/zh_cn.lang]: 伪金块

**Comment by Drullkus** [comment]
Excellent, thanks for using the 1.12 branch :)

Will wait for other guy before merge

**Comment by liach** [comment]
Minoshroom is not Minotaur. I suggest calling it `米诺菇`.

File: src/main/resources/assets/twilightforest/lang/zh_cn.lang [hunk 2]
```diff
@@ -239,57 +384,39 @@ entity.twilightforest:kobold.name=哥布林
...
 entity.twilightforest:lich_minion.name=巫妖的仆从
 entity.twilightforest:loyal_zombie.name=忠诚的僵尸
-entity.twilightforest:mosquito_swarm.name=成群的蚊子
-entity.twilightforest:death_tome.name=死神之书
-entity.twilightforest:minotaur.name=牛头怪
-entity.twilightforest:minoshroom.name=米陶诺
+entity.twilightforest:mosquito_swarm.name=蚊群
+entity.twilightforest:death_tome.name=死灵书
+entity.twilightforest:minotaur.name=牛头人
+entity.twilightforest:minoshroom.name=米诺陶
```
Comment (liach) [inline src/main/resources/assets/twilightforest/lang/zh_cn.lang]: here

**Comment by 3TUSK** [comment]
Danm.. Two translations swapped their seats...

**Review by TartaricAcid — APPROVED** [review]

File: src/main/resources/assets/twilightforest/lang/zh_cn.lang [hunk 3]
```diff
@@ -277,7 +283,7 @@ item.steeleafShovel.name=钢叶铲
 item.steeleafPick.name=钢叶镐
 item.steeleafAxe.name=钢叶斧
 item.steeleafHoe.name=钢叶锄
-item.minotaurAxe.name=米诺陶诺斯战斧
+item.minotaurAxe.name=米诺陶战斧
 item.mazebreakerPick.name=迷宫破坏者
 item.transformPowder.name=转换粉
 item.meefRaw.name=生牛头人肉
```
Comment (liach) [inline src/main/resources/assets/twilightforest/lang/zh_cn.lang]: Do you need to fix this, too?

Comment (3TUSK) [reply src/main/resources/assets/twilightforest/lang/zh_cn.lang]: Isn't meef == minotaur + beef?

Comment (Drullkus) [reply src/main/resources/assets/twilightforest/lang/zh_cn.lang]: Yes. Alternatively it could also be man+beef. Up to you!

Comment (liach) [reply src/main/resources/assets/twilightforest/lang/zh_cn.lang]: I thought it'd be minoshroom's meat.

**Review by liach — APPROVED** [review]

File: src/main/resources/assets/twilightforest/lang/zh_cn.lang (hunk 3 above)
Comment (3TUSK) [reply src/main/resources/assets/twilightforest/lang/zh_cn.lang]: Seems like you @liach forget the fact that minoshroom drops stroganoff (also the minotaur axe), which... corresponding to the mushroom stew.

Comment (liach) [reply src/main/resources/assets/twilightforest/lang/zh_cn.lang]: eh, I haven't played tf a lot, in fact.

**Comment by ETStareak** [comment]
Checked.It seems nothing uncorrect.I think it is great.

**Comment by liach** [comment]
"uncorrect" -> "incorrect"
"nothing is incorrect" instead of "nothing uncorrect'

**Comment by 3TUSK** [comment]
> Me fail English? That's unpossible!

@ETStarLight https://github.com/TeamTwilight/twilightforest/pull/143/files 右上角 "Review Changes" -> "Approval"

**Review by ETStareak — APPROVED** [review]

**Comment by 3TUSK** [comment]
@Drullkus Ready to go.

**Comment by Drullkus** [comment]
Thank you for your help everyone! :)

**Comment by 3TUSK** [comment]
uh, one more question: is 1.11 branch no longer supported in favor of 1.12.1 branch?

**Comment by williewillus** [comment]
correct

**Comment by 3TUSK** [comment]
*oh yeah i have reason for not doing translation for 1.11*

**Pull request merged**